src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1937,19 +1929,33 @@ public enum RisingRivals implements LogicCardInfo {
               damage 30
               afterDamage {
                 targeted (defending) {
-                  bc "During ${opp.owner.getPlayerUsername(bg)}'s next turn, Energy can't be attached from their hand to the Defending ${defending}. (This effect can be removed by evolving or benching ${defending}.)"
+                  bc "During ${opp.owner.getPlayerUsername(bg)}'s next turn, ${opp.owner.getPlayerUsername(bg)} can't play any Pokémon from their hand to Evolve or Level Up the defending $defending. (This effect can be removed by evolving or benching ${defending}.)"
```

Comment:
Is this broadcast needed since a warning explaining the issue pops up when the affected player tries to play a relevant card?

Reply:
I like it personally for two reasons. Firstly it makes the effect more visible if it is ever involved in a bug report, and it makes scenarios where a player will search their deck for a card they can't play less likely.

Reply:
Hmm, I won't argue against it for now. Maybe when we get effect targeting in we can also add a list of effects affecting the player and their Pokémon where we'd put descriptions like these instead.

I don't think it'd come up in bug reports. The warning would make it pretty obvious why it was prevented, and if the reporter describes it well enough or we see the move that was used near the end of the log we'd be able to deduce the "issue" pretty quick.